data/saas/config/universal_analytics_config.yml

```diff
@@ -0,0 +1,111 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Universal Analytics (UA) SaaS Config
+  type: universal_analytics
+  description: A schema representing the Universal Analytics connector for Fides
+  version: 0.0.1
+
+  connector_params:
+
+    - name: client_id
+    - name: client_secret
+    - name: redirect_uri
+    - name: web_property_id
```

Comment:
Setup screens are nearly identical:

![Screen Shot 2023-02-01 at 4 02 31 PM](https://user-images.githubusercontent.com/9755598/216173618-dee7f1c4-88f7-4e21-9c3d-7308ff7709f0.png)

![Screen Shot 2023-02-01 at 4 02 23 PM](https://user-images.githubusercontent.com/9755598/216173634-05ae67b3-3fbe-4c00-8934-840813896d44.png)

Reply:
That should be okay. 
Can we change the "Google Analytics" text to "Google Analytics 4"?

Reply:
Sure can!

Reply:
@saiatethyca, Google Analytics 4 human readable name updated:
<img width="1176" alt="Screen Shot 2023-02-01 at 4 52 30 PM" src="https://user-images.githubusercontent.com/9755598/216189886-37c8d1f6-e30a-4f23-8658-ebd8ebc9f63f.png">